projects/client-api-react/src/stories/Graphistry.stories.jsx

```diff
@@ -33,6 +35,31 @@ export const OnClientAPIConnected = (args) => {
   </div>);
 }
 
+export const OnSelectionUpdate = (args) => {
+  const [ selection, setSelection ] = useState(undefined);
```

Comment:
good to show what the sample return is (nodes, edges, their values) so dev gets to see the schema